### ISSUE TITLE
Is there any way I can test using CPU

### ISSUE THREAD

**Issue by Songloading** [CSAILVision/semantic-segmentation-pytorch#208]
Several issues here. 
After I torch.cuda.set_device(-1)
segmentation_module.cuda() and test(segmentation_module, loader_test, -1) do not perform well.
Is there any way I can test using CPU?

**Comment by joshmyersdean** [comment]
I would like to know this as well!

**Comment by aminsabetsarvestani** [comment]
I would like to run this on CPU as well

**Comment by wwww666** [comment]
So,did you solve it?

**Comment by nelamon** [comment]
@Songloading Just curious to know if you were able to test it using CPU?

**Comment by RodMech** [comment]
Yeah, you can test it in CPU, performance in an i7 2,2 GHz is 19.14 s / it

**Comment by tejaswigowda** [comment]
Please help

**Comment by tejaswigowda** [comment]
It works!

**Comment by MrPhilosopher** [comment]
@tejaswigowda  @RodMech  can you guys help in testing via CPU ?
